parsing/parser.mly

```diff
@@ -238,7 +238,7 @@ let ldot x y = Ldot(x,y)
 let dotop_fun ~loc dotop =
   (* We could use ghexp here, but sticking to mkexp for parser.mly
      compatibility. TODO improve parser.mly *)
```

Comment:
Is this comment now outdated?

Reply:
It is indeed, I just pushed a commit removing it.
Thanks.